apple/Tests/TestCases/Profile/DeviceInfoTests.swift

```diff
@@ -76,6 +76,50 @@ final class DeviceInfoTests: Test<DeviceInfo> {
         }
         """, expected: nil)
 	}
+	
+	func test_date() {
+		
+		/// This matches `[bindings.swift.custom_types.Timestamp]`
+		/// inside `uniffi.toml` with the only difference that we use `$0` here but
+		/// the toml uses `{}`;
+		let into_custom: (String) -> Date = {let s = $0;let df = ISO8601DateFormatter();let fo = ISO8601DateFormatter.Options.withFractionalSeconds;df.formatOptions.insert(fo);func t() -> Date? {df.date(from: s)};if let d = t() {return d};df.formatOptions.remove(fo);return t()!}
+		
+		/// This matches `[bindings.swift.custom_types.Timestamp]`
+		/// inside `uniffi.toml`
+		let from_custom: (Date) -> String = { let df = ISO8601DateFormatter();df.formatOptions.insert(.withFractionalSeconds);return df.string(from: $0) }
```

Comment:
Could just be `$0.formatted(.iso8601.time(includingFractionalSeconds: true))`

Reply:
nope, this produces *only* time - `17:13:56.123`.